packages/react-text/src/components/Body/Body.tsx

```diff
@@ -1,5 +1,5 @@
 import { makeStyles } from '@fluentui/react-make-styles';
-import { typographyStyles } from '../../index';
+import { typographyStyles } from '../../typographyStyles/index';
```

Comment:
so we will not provide typography styles as public API ?

Reply:
Nope, outcome was to not export until required. Wrappers are public but, until required, we won't export the styles.